src/plugins/links/public/components/editor/link_editor.tsx

```diff
@@ -108,7 +108,7 @@ export const LinkEditor = ({
           </EuiTitle>
         </EuiButtonEmpty>
       </EuiFlyoutHeader>
-      <EuiFlyoutBody>
+      <EuiFlyoutBody data-test-subj="links--linkEditor--flyout">
```

Comment:
The `data-test-subj` on the focus trap didn't actually get attached to any element - so, I moved it to the flyout body so that we can actually check when the flyout is open.